README-OSX.md

```diff
@@ -0,0 +1,27 @@
+## Developing on OSX
```

Comment:
I believe this is already in the repo under a slightly different name?

Reply:
Yeah, thanks. Forgot I renamed it and this is left over.